ykman/descriptor.py

```diff
@@ -153,9 +154,13 @@ def get_descriptors():
 
 def _list_drivers(transports):
     if TRANSPORT.CCID & transports:
-        for dev in open_ccid():
-            if dev:
-                yield dev
+        try:
+            for dev in open_ccid():
+                if dev:
+                    yield dev
+        except smartcard.pcsc.PCSCExceptions.EstablishContextException:
+            logger.debug('Failed to establish CCID context. '
+                         'Is the pcscd service running?')
```

Comment:
maybe we should say: `Is pcscd/smart card services running?` since it's not called pcscd on windows.